### PR TITLE
Lwt 2.5.0

### PR DESCRIPTION
**Pull request by vbmithr**

### CODE REVIEW

File: packages/lwt/lwt.2.5.0/opam [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    "--%{conf-libev:enable}%-libev"
+    "--%{camlp4:enable}%-camlp4"
+    "--%{react:enable}%-react"
+    "--%{ssl:enable}%-ssl"
+    "--%{base-unix:enable}%-unix"
+    "--%{base-threads:enable}%-preemptive"
+    "--%{lablgtk:enable}%-glib"
+    "--%{ppx_tools:enable}%-ppx"]
+  [make "build"]
+  [make "install"]
```
Comment (dsheets) [inline packages/lwt/lwt.2.5.0/opam]: Could you move this to `install`, please? It will improve parallelizability of builds.

**Comment by dsheets** [comment]
Also, a PR which sets upper bounds on lwt dependent libraries that use `Lwt_io` functions with `~buffer_size` would be much appreciated before this gets merged.

**Comment by avsm** [comment]
This one will also need upper bounds on existing consumers of Lwt due to the `buffer_size` API change before it can be merged.

File: packages/lwt/lwt.2.5.0/opam [hunk 2]
```diff
@@ -0,0 +1,43 @@
+opam-version: "1.2"
+maintainer: "Jérôme Vouillon <jerome.vouillon@pps.univ-paris-diderot.fr>"
```
Comment (Chris00) [inline packages/lwt/lwt.2.5.0/opam]: This should be added to `_oasis`.

File: packages/lwt/lwt.2.5.0/opam [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+authors: [
+  "Jérôme Vouillon"
+  "Jérémie Dimino"
+]
+homepage: "http://github.com/ocsigen/lwt"
+dev-repo: "git://github.com/ocsigen/lwt"
```
Comment (Chris00) [inline packages/lwt/lwt.2.5.0/opam]: Please update the repo in `_oasis`.

File: packages/lwt/lwt.2.5.0/opam [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    "--%{base-unix:enable}%-unix"
+    "--%{base-threads:enable}%-preemptive"
+    "--%{lablgtk:enable}%-glib"
+    "--%{ppx_tools:enable}%-ppx"]
+  [make "build"]
+]
+install: [[make "install"]]
+remove: [["ocamlfind" "remove" "lwt"]]
+depends: [
+  "ocamlfind" {>= "1.5.0"}
```
Comment (Chris00) [inline packages/lwt/lwt.2.5.0/opam]: You may want to say ` "ocamlfind" {build & >= "1.5"}`.

**Comment by dsheets** [comment]
#4390 was as good as we can manage automatically. Merging now and waiting for build failure reports.

**Pull request merged**